cargo-dist/src/backend/installer/homebrew.rs

```diff
@@ -265,4 +324,40 @@ mod tests {
     fn ampersand_but_no_digit() {
         run_comparison("openssl@blah", "Openssl@blah");
     }
+
+    fn run_spdx_comparison(spdx_string: &str, homebrew_dsl: &str) {
```

Comment:
We can probably tag this with `#[test]` too since it doesn't need to be compiled into the production binary.

Reply:
I was following the example of the `run_comparison()` function in the same test module, my assumption was the whole module gets excluded because of the `#[cfg(test)]`. Should I tag `run_comparison()` as well?

Reply:
No, you had it right - I missed this was inside the test module.